lib/logstash/inputs/kafka.rb

```diff
@@ -98,7 +104,7 @@ def register
       options[:reset_beginning] = 'from-beginning'
     end # if :reset_beginning
     @kafka_client_queue = SizedQueue.new(@queue_size)
-    @consumer_group = Kafka::Group.new(options)
+    @consumer_group = @kafka_group_class.new(options)
```

Comment:
No you're not, both solutions are ok! I would like us to come to an style something we like.